packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -3227,6 +3227,9 @@ export class WalletService {
    * @returns {TxProposal[]} Transaction proposal.
    */
   getPendingTxs(opts, cb) {
+    if (opts.tokenAddress) {
```

Comment:
why tokenAddress can't have pending tx?

Reply:
Because token wallets are duplicate ETH wallets. So anytime we make a `txp`, there will be two duplicate `txps`. `walletEth.pendingTxs`

Reply:
token wallets do not get stored on the BWS/Node wallets collection so txps cannot get shared between devices for token wallets